komga/src/main/kotlin/org/gotson/komga/domain/service/SeriesLifecycle.kt

```diff
@@ -240,15 +246,61 @@ class SeriesLifecycle(
       thumbnailsSeriesRepository.markSelected(thumbnail)
   }
 
+  fun getAllThumbnailsForSeries(seriesId: String): Collection<ThumbnailSeries> {
+    return thumbnailsSeriesRepository.findAllBySeriesId(seriesId)
+  }
```

Comment:
for such methods where no logic is done, you can use the repository directly instead of wrapping it here